pt/core-libraries/email.rst

```diff
@@ -1,13 +1,316 @@
+
+Definição de perfis de entrega permitem consolidar as configurações de e-mail comuns em perfis reutilizáveis. Seu aplicativo 
+pode ter tantos perfis como necessário. As seguintes chaves de configuração são usados:
+
+- ``'from'``: E-mail ou array do remetente. Visto ``Email::from()``.
+- ``'sender'``: E-mail ou array do Remetente original. Visto ``Email::sender()``.
+- ``'to'``: E-mail ou array do Destinatário. Visto ``Email::to()``.
+- ``'cc'``: E-mail ou array da Copia de Carbono. Visto ``Email::cc()``.
+- ``'bcc'``: E-mail ou array da cópia oculta. Visto ``Email::bcc()``.
+- ``'replyTo'``: Email ou array do E-mail de respostas. Visto ``Email::replyTo()``.
+- ``'readReceipt'``: Endereço de E-mail ou array de endereços para receber a recepção de leitura. Visto ``Email::readReceipt()``.
+- ``'returnPath'``: Endereço de E-mail ou um array de endereços para retornar se teve alguns erros. Visto ``Email::returnPath()``.
+- ``'messageId'``: ID da mensagem do e-mail. Visto ``Email::messageId()``.
+- ``'subject'``: Assunto da mensagem. Visto ``Email::subject()``.
+- ``'message'``: Conteúdo de mensagem. Não defina este campo se você estiver usando o conteúdo processado.
+- ``'headers'``: Cabeçalhos sejam incluídas. Visto ``Email::setHeaders()``.
+- ``'viewRender'``: Se você estiver usando conteúdo renderizado, definir o nome da classe da view.
+  Visto ``Email::viewRender()``.
+- ``'template'``: Se você estiver usando conteúdo renderizado, definir o nome do template. Visto
+  ``Email::template()``.
+- ``'theme'``: Tema usado quando o template é renderizado. Visto ``Email::theme()``.
+- ``'layout'``: Se você estiver usando conteúdo renderizado, definir o layout para renderizar. Se você quer renderizar um template sem layout, definir este campo como null. Visto ``Email::template()``.
+- ``'viewVars'``: Se você estiver usando conteúdo renderizado, definir o array com as variáveis para serem usadas na view. Visto ``Email::viewVars()``.
+- ``'attachments'``: Lista de arquivos para anexar. Visto ``Email::attachments()``.
+- ``'emailFormat'``: Formato do e-mail (html, text ou both). Visto ``Email::emailFormat()``.
+- ``'transport'``: Nome da configuração de transporte. Visto
+  :php:meth:`~Cake\\Mailer\\Email::configTransport()`.
+- ``'log'``: Nível de log para registrar os cabeçalhos de e-mail e mensagem. ``true`` usará
+  LOG_DEBUG. Visto tabmém como ``CakeLog::write()``
+- ``'helpers'``: Array de helpers usado no template do e-mail.
+
+Todas essas configurações são opcionais, exceto ``'from'``.
+
 .. note::
-    A documentação não é atualmente suportada pela lingua portuguesa nesta
-    página.
 
-    Por favor, sinta-se a vontade para nos enviar um pull request no
-    `Github <https://github.com/cakephp/docs>`_ ou use o botão
-    **Improve This Doc** para propor suas mudanças diretamente.
+Os valores das chaves acima usando e-mail ou array, como from, to, cc, etc será passado como primeiro parâmetro de métodos correspondentes. O equivalente de:
```

Comment:
You should indent this line for the note to be displayed correctly. It will solve your build warning.